Survey/Sources/Presentation/Views/AnswerViews/NPS/NPSAnswerView.swift

```diff
@@ -50,24 +73,16 @@ struct NPSAnswerView: View {
 
     private func setUpDescription() -> some View {
         HStack {
-            Text("Not at all Likely")
+            Text(AssetLocalization.npsNotLikelyText())
+                .opacity((rating < 5) && (rating >= 0) ? 1.0 : 0.5)
                 .foregroundColor(.white)
                 .font(.boldBody)
             Spacer()
-            Text("Extremely Likely")
+            Text(AssetLocalization.npsLikelyText())
+                .opacity(rating >= 5 ? 1.0 : 0.5)
```

Comment:
Same for this.

Reply:
I fixed at https://github.com/nkhanh44/ic-survey-ios/pull/78/commits/0aa8d9464b34c2d7406787934d41b70938be2ada.